### ISSUE TITLE
Input not recognized

### ISSUE THREAD

**Issue by zach-morris** [kodi-game/game.libretro.stella#3]
Version tested:  RetroPlayer 18 alpha1 2018-02-11

After mapping all the buttons for game.controller.atari.2600, and launching a game in Stella (Pitfall), the only input that seems to be recognized was the reset button.  In the log below I press fire, right a few times and then reset.

```
21:38:47.562 T:123145314799616   DEBUG: FEATURE [ a ] on game.controller.default released
21:38:47.563 T:123145314799616   DEBUG: FEATURE [ fire ] on game.controller.atari.2600 released
21:38:47.671 T:123145314799616   DEBUG: BUTTON [ 14 ] on "PS(R) Gamepad" pressed
21:38:47.672 T:123145314799616   DEBUG: FEATURE [ a ] on game.controller.default pressed (ignored)
21:38:47.672 T:123145314799616   DEBUG: FEATURE [ fire ] on game.controller.atari.2600 pressed (handled)
21:38:47.774 T:123145314799616   DEBUG: BUTTON [ 14 ] on "PS(R) Gamepad" released
21:38:47.774 T:123145314799616   DEBUG: FEATURE [ a ] on game.controller.default released
21:38:47.775 T:123145314799616   DEBUG: FEATURE [ fire ] on game.controller.atari.2600 released
21:38:47.864 T:123145314799616   DEBUG: BUTTON [ 14 ] on "PS(R) Gamepad" pressed
21:38:47.864 T:123145314799616   DEBUG: FEATURE [ a ] on game.controller.default pressed (ignored)
21:38:47.864 T:123145314799616   DEBUG: FEATURE [ fire ] on game.controller.atari.2600 pressed (handled)
21:38:47.972 T:123145314799616   DEBUG: BUTTON [ 14 ] on "PS(R) Gamepad" released
21:38:47.972 T:123145314799616   DEBUG: FEATURE [ a ] on game.controller.default released
21:38:47.972 T:123145314799616   DEBUG: FEATURE [ fire ] on game.controller.atari.2600 released
21:38:48.366 T:123145314799616   DEBUG: BUTTON [ 3 ] on "PS(R) Gamepad" pressed
21:38:48.367 T:123145314799616   DEBUG: FEATURE [ start ] on game.controller.default pressed (ignored)
21:38:48.367 T:123145314799616   DEBUG: FEATURE [ start ] on game.controller.default pressed (handled)
21:38:48.440 T:123145314799616   DEBUG: BUTTON [ 3 ] on "PS(R) Gamepad" released
21:38:48.440 T:123145314799616   DEBUG: FEATURE [ start ] on game.controller.default released
21:38:48.819 T:123145314799616   DEBUG: Previous line repeats 1 times.
21:38:48.819 T:123145314799616   DEBUG: BUTTON [ 3 ] on "PS(R) Gamepad" pressed
21:38:48.819 T:123145314799616   DEBUG: FEATURE [ start ] on game.controller.default pressed (ignored)
21:38:48.820 T:123145314799616   DEBUG: FEATURE [ start ] on game.controller.default pressed (handled)
21:38:48.872 T:123145314799616   DEBUG: BUTTON [ 3 ] on "PS(R) Gamepad" released
21:38:48.872 T:123145314799616   DEBUG: FEATURE [ start ] on game.controller.default released
21:38:49.014 T:123145314799616   DEBUG: Previous line repeats 1 times.
21:38:49.014 T:123145314799616   DEBUG: BUTTON [ 3 ] on "PS(R) Gamepad" pressed
21:38:49.014 T:123145314799616   DEBUG: FEATURE [ start ] on game.controller.default pressed (ignored)
21:38:49.014 T:123145314799616   DEBUG: FEATURE [ start ] on game.controller.default pressed (handled)
21:38:49.055 T:123145320701952   DEBUG: CAddonSettings[service.nextup.notification]: loading setting definitions
21:38:49.055 T:123145320701952   DEBUG: CAddonSettings[service.nextup.notification]: trying to load setting definitions from old format...
21:38:49.071 T:123145314799616   DEBUG: BUTTON [ 3 ] on "PS(R) Gamepad" released
21:38:49.071 T:123145314799616   DEBUG: FEATURE [ start ] on game.controller.default released
21:38:49.191 T:123145314799616   DEBUG: Previous line repeats 1 times.
21:38:49.191 T:123145314799616   DEBUG: BUTTON [ 3 ] on "PS(R) Gamepad" pressed
21:38:49.191 T:123145314799616   DEBUG: FEATURE [ start ] on game.controller.default pressed (ignored)
21:38:49.191 T:123145314799616   DEBUG: FEATURE [ start ] on game.controller.default pressed (handled)
21:38:49.276 T:123145314799616   DEBUG: BUTTON [ 3 ] on "PS(R) Gamepad" released
21:38:49.276 T:123145314799616   DEBUG: FEATURE [ start ] on game.controller.default released
21:38:49.617 T:123145314799616   DEBUG: Previous line repeats 1 times.
21:38:49.617 T:123145314799616   DEBUG: BUTTON [ 0 ] on "PS(R) Gamepad" pressed
21:38:49.617 T:123145314799616   DEBUG: FEATURE [ back ] on game.controller.default pressed (ignored)
21:38:49.617 T:123145314799616   DEBUG: FEATURE [ select ] on game.controller.atari.2600 pressed (handled)
21:38:49.725 T:123145314799616   DEBUG: BUTTON [ 0 ] on "PS(R) Gamepad" released
21:38:49.725 T:123145314799616   DEBUG: FEATURE [ back ] on game.controller.default released
21:38:49.725 T:123145314799616   DEBUG: FEATURE [ select ] on game.controller.atari.2600 released
21:38:49.906 T:123145314799616   DEBUG: BUTTON [ 0 ] on "PS(R) Gamepad" pressed
21:38:49.906 T:123145314799616   DEBUG: FEATURE [ back ] on game.controller.default pressed (ignored)
21:38:49.906 T:123145314799616   DEBUG: FEATURE [ select ] on game.controller.atari.2600 pressed (handled)
21:38:49.992 T:123145314799616   DEBUG: BUTTON [ 0 ] on "PS(R) Gamepad" released
21:38:49.992 T:123145314799616   DEBUG: FEATURE [ back ] on game.controller.default released
21:38:49.992 T:123145314799616   DEBUG: FEATURE [ select ] on game.controller.atari.2600 released
21:38:50.156 T:123145314799616   DEBUG: BUTTON [ 0 ] on "PS(R) Gamepad" pressed
21:38:50.156 T:123145314799616   DEBUG: FEATURE [ back ] on game.controller.default pressed (ignored)
21:38:50.157 T:123145314799616   DEBUG: FEATURE [ select ] on game.controller.atari.2600 pressed (handled)
21:38:50.224 T:123145314799616   DEBUG: BUTTON [ 0 ] on "PS(R) Gamepad" released
21:38:50.225 T:123145314799616   DEBUG: FEATURE [ back ] on game.controller.default released
21:38:50.225 T:123145314799616   DEBUG: FEATURE [ select ] on game.controller.atari.2600 released
21:38:50.352 T:123145314799616   DEBUG: BUTTON [ 0 ] on "PS(R) Gamepad" pressed
21:38:50.352 T:123145314799616   DEBUG: FEATURE [ back ] on game.controller.default pressed (ignored)
21:38:50.352 T:123145314799616   DEBUG: FEATURE [ select ] on game.controller.atari.2600 pressed (handled)
21:38:50.423 T:123145314799616   DEBUG: BUTTON [ 0 ] on "PS(R) Gamepad" released
21:38:50.423 T:123145314799616   DEBUG: FEATURE [ back ] on game.controller.default released
21:38:50.423 T:123145314799616   DEBUG: FEATURE [ select ] on game.controller.atari.2600 released
21:38:50.566 T:123145314799616   DEBUG: BUTTON [ 0 ] on "PS(R) Gamepad" pressed
21:38:50.566 T:123145314799616   DEBUG: FEATURE [ back ] on game.controller.default pressed (ignored)
21:38:50.567 T:123145314799616   DEBUG: FEATURE [ select ] on game.controller.atari.2600 pressed (handled)
21:38:50.656 T:123145314799616   DEBUG: BUTTON [ 0 ] on "PS(R) Gamepad" released
21:38:50.656 T:123145314799616   DEBUG: FEATURE [ back ] on game.controller.default released
21:38:50.657 T:123145314799616   DEBUG: FEATURE [ select ] on game.controller.atari.2600 released
21:38:52.320 T:123145314799616   DEBUG: BUTTON [ 1 ] on "PS(R) Gamepad" pressed
21:38:52.320 T:123145314799616   DEBUG: FEATURE [ leftthumb ] on game.controller.default pressed (ignored)
21:38:52.320 T:123145314799616   DEBUG: FEATURE [ reset ] on game.controller.atari.2600 pressed (handled)
21:38:52.442 T:123145314799616   DEBUG: BUTTON [ 1 ] on "PS(R) Gamepad" released
21:38:52.442 T:123145314799616   DEBUG: FEATURE [ leftthumb ] on game.controller.default released
21:38:52.442 T:123145314799616   DEBUG: FEATURE [ reset ] on game.controller.atari.2600 released
```

**Comment by garbear** [comment]
Fixed by https://github.com/kodi-game/game.libretro.stella/pull/4 and https://github.com/kodi-game/kodi-game-controllers/pull/22. I'll include these in my next build for you to confirm.

**Comment by zach-morris** [comment]
Confirmed fixed in 18.0-ALPHA1 Git:20180215-276e1a4
